Minor readability fixes and Policy Definitions note. (#1412)
MicrosoftDocs/OfficeDocs-SharePoint: OneDrive/use-group-policy.md

  
3. Copy the .adml and .admx files.

> 4. Paste the .admx file in your domain's Central Store, \\*domain*\sysvol\domain\Policies\PolicyDefinitions (where *domain* is your domain name, such as corp.contoso.com), and the .adml in the appropriate language subfolder (such as en-us). If the PolicyDefinitions folder does not exist, see [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/en-us/help/3087759) or use your local policy store under %windir%\policydefinitions.

Reviewer: ```suggestion
4. Paste the .admx file in your domain's Central Store, \\*domain*\sysvol\domain\Policies\PolicyDefinitions (where *domain* is your domain name, such as corp.contoso.com), and the .adml in the appropriate language subfolder (such as en-us). If the PolicyDefinitions folder does not exist, see [How to create and manage the Central Store for Group Policy Administrative Templates in Windows](https://support.microsoft.com/help/3087759) or use your local policy store under %windir%\policydefinitions.
```

  
[HKLM\SOFTWARE\Policies\Microsoft\OneDrive]"FilesOnDemandEnabled"="dword:00000001"
 

> Meet Windows and OneDrive sync client requirements and still can't see Files On-Demand option available at "Settings"? Make sure the service "Windows Cloud Files Filter Driver" start type is set to 2 (AUTO_START). Enabling this feature sets the following registry key value to 2.

Reviewer: ```suggestion
Are you meeting Windows and OneDrive sync client requirements and still can't see the Files On-Demand option available at "Settings"? Make sure the service "Windows Cloud Files Filter Driver" start type is set to 2 (AUTO_START). Enabling this feature sets the following registry key value to 2.
```

![The Getting ready to sync dialog box](media/copy-library-id.png)

> [!NOTE]

> > The special characters in this copied string are in Unicode which needs to be converted to ASCII find and replace the following:

Reviewer: ```suggestion
> The special characters in this copied string are in Unicode which needs to be converted to ASCII; find and replace the following:
```
